G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/Commands/SeleniumAlertActionCommand.md

```diff
@@ -0,0 +1,35 @@
+# selenium.alertaction
+
+## Syntax
+
+```G1ANT
+selenium.alertaction action ⟦text⟧
+```
+
+## Description
+
+Waits for an alert and performs a specified action.
+
+| Argument       | Type                                                         | Required | Default Value                                                | Description                                                  |
+| -------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `action`       | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | yes      |                                                              | Specifies an action: `accept` or `dismiss`                   |
+| `if`           | [bool](G1ANT.Language/G1ANT.Language/Structures/BooleanStructure.md) | no       | true                                                         | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](G1ANT.Language/G1ANT.Language/Structures/TimeSpanStructure.md) | no       | [♥timeoutselenium](G1ANT.Language/G1ANT.Addon.Core/Variables/TimeoutSeleniumVariable.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](G1ANT.Language/G1ANT.Language/Structures/ProcedureStructure.md) | no       |                                                              | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](G1ANT.Language/G1ANT.Language/Structures/LabelStructure.md) | no       |                                                              | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no       |                                                              | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [variable](G1ANT.Language/G1ANT.Language/Structures/VariableStructure.md) | no       |                                                              | Name of a variable that will store the returned exception. The variable will be of [error](G1ANT.Language/G1ANT.Language/Structures/ErrorStructure.md) structure |
+
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](G1ANT.Manual/appendices/common-arguments.md) page.
+
+## Example
+
+In the following example, the robot opens Chrome, runs a Java script, which brings up a confirmation message and after a 2-second delay, confirms this message (equivalent of clicking OK):
```

Comment:
it's not a Java script. Java and Javascript are different languages and here it is a Javascript script 😄

Reply:
I'm aware of that. One space too many.

Reply:
```suggestion
In the following example, the robot opens Chrome, runs a Javascript, which brings up a confirmation message and after a 2-second delay, confirms this message (equivalent of clicking OK):
```

Reply:
Yeah, I saw later that everywhere where there was a Javascript code, you explained it correctly :) But I forgot so I left my first comment unchanged, I know it was just a typo 😄